docs/src/components/HeadCommon.astro

```diff
@@ -0,0 +1,49 @@
+---
+import '../styles/theme.css';
+import '../styles/index.css';
+---
+
+<!-- Global Metadata -->
+<meta charset="utf-8" />
+<meta name="viewport" content="width=device-width" />
+<meta name="generator" content={Astro.generator} />
+
+<link rel="icon" sizes="16x16" href="/favicon/android-icon-16x16.png" />
+<link rel="icon" sizes="32x32" href="/favicon/android-icon-32x32.png" />
+<link rel="icon" sizes="96x96" href="/favicon/favicon-96x96.png" />
+
+<link rel="sitemap" href="/sitemap.xml" />
+
+<!-- Preload Fonts -->
+<link rel="preconnect" href="https://fonts.googleapis.com" />
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+<link
+  href="https://fonts.googleapis.com/css2?family=IBM+Plex+Mono:ital,wght@0,400;0,700;1,400&display=swap"
+  rel="stylesheet"
+/>
+
+<!-- Scrollable a11y code helper -->
+<script src="/make-scrollable-code-focusable.js" is:inline></script>
```

Comment:
not-blocking: this seems to be running too early, so, pre elements aren't getting `tabindex` applied to them.
Tested on https://media-chrome-docs-git-fork-luwes-docs-site-mux.vercel.app/en/media-controller#styling

Reply:
```suggestion
<script src="/make-scrollable-code-focusable.js" is:inline defer></script>
```

Deferring the script seems to fix it.